core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -978,7 +979,7 @@ class KafkaConfig(val props: java.util.Map[_, _], doLog: Boolean) extends Abstra
       s"${KafkaConfig.AdvertisedListenersProp} protocols must be equal to or a subset of ${KafkaConfig.ListenersProp} protocols. " +
       s"Found ${advertisedListeners.keySet}. The valid options based on currently configured protocols are ${listeners.keySet}"
     )
-    require(interBrokerProtocolVersion.onOrAfter(ApiVersion(messageFormatVersion)),
+    require(interBrokerProtocolVersion >= messageFormatVersion,
       s"message.format.version $messageFormatVersion cannot be used when inter.broker.protocol.version is set to $interBrokerProtocolVersion")
```

Comment:
Yes, if someone sets interBrokerProtocolVersion or MessageFormatVersion to 0.10.0-IV0, it makes sense to log 0.10.0-IV0. It's just that if someone sets it to 0.10.0, we still log 0.10.0-IV0. This is due to the implementation in KIP-32 where we share the case object btw 0.10.0 and 0.10.0-IV0. I am just wondering if we can make the logging clearer.